.gitignore

```diff
@@ -5,8 +5,14 @@
 !LICENSE
 !README.md
 
+
+
```

Comment:
Did your editor put these in?

Reply:
Probably, I have a make things pretty on my editor so it may have done that